src/journeys_utils.js

```diff
@@ -594,7 +652,8 @@ journeys_utils._getPageviewMetadata = function(options, additionalMetadata) {
 		"user_agent": navigator.userAgent,
 		"language": navigator.language,
 		"screen_width": screen.width || -1,
-		"screen_height": screen.height || -1
+		"screen_height": screen.height || -1,
+		"window_device_pixel_ratio": window.devicePixelRatio || 1
```

Comment:
Also passing the screen pixel ration to calculate the device screen size in the backend to calculate the logic for safe area flag.